framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -2625,7 +2642,7 @@ public boolean shouldSkipUses(@FullyQualifiedName String typeName) {
      * @param tree class to potentially skip
      * @return true if checker should not test {@code tree}
      */
-    public final boolean shouldSkipDefs(ClassTree tree) {
+    public boolean shouldSkipDefs(ClassTree tree) {
```

Comment:
Instead of making these two methods non-final, how about adding handling of `parentChecker` to these methods?
The logic in `NullnessNoInitSubchecker.java` makes sense for all checkers with a parent, right?

Reply:
Will be addressed by #572